themes/buildpacks/assets/scss/_syntax.scss

```diff
@@ -1,5 +1,5 @@
 /* Background */ .chroma { background-color: #f0f3f3 }
-/* Error */ .chroma .err { color: #aa0000; background-color: #ffaaaa }
+// /* Error */ .chroma .err { color: #aa0000; background-color: #ffaaaa }  // Eventually, fix lexer rather than remove this
```

Comment:
This is to prevent an `err` label being generated and being highlighted on toml.

**Example:**

```
run-images: [ ]
```

The `-` would be highlighted.

Ref: https://github.com/alecthomas/chroma/blob/master/lexers/t/toml.go